tasks/main.yml

```
@@ -1,2 +1,16 @@
 ---
-# tasks file for skeleton
+- name: Create the /var/cyhy/admiral directory
+  ansible.builtin.file:
+    group: "{{ admiral_file_owner_group | default(omit) }}"
+    mode: "0o755"
```

We have been using a different syntax for `mode` for a while now: 
```suggestion
    mode: u=rwx,g=rx,o=rx
```